### PR TITLE
2 packages from abenori/satysfi-pagenumber at 1.0.0

### PR DESCRIPTION
**Pull request by abenori**
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-pagenumber-doc.1.0.0 satysfi-pagenumber.1.0.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-5` :: satysfi-pagenumber-doc.1.0.0 satysfi-pagenumber.1.0.0
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-pagenumber-doc.1.0.0 satysfi-pagenumber.1.0.0

### CODE REVIEW

**Comment by na4zagin3** [comment]
Let me reopen this PR to rerun the GitHub Actions.

File: packages/satysfi-pagenumber-doc/satysfi-pagenumber-doc.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Brilliant description comes here.
+"""
+maintainer: "Noriyuki Abe"
+authors: "Noriyuki Abe"
+license: "BSD-2-Clause-FreeBSD"
+homepage: "https://github.com/abenori/satysfi-pagenumber"
+dev-repo: "git+https://github.com/abenori/satysfi-pagenumber.git"
+bug-reports: "https://github.com/abenori/satysfi-pagenumber/issues"
+depends: [
+  "satysfi" { >= "0.0.5" }
```
Comment (na4zagin3) [inline packages/satysfi-pagenumber-doc/satysfi-pagenumber-doc.1.0.0/opam:13]: Can you specify the upper bound?

```suggestion
  "satysfi" { >= "0.0.5" & < "0.0.7"}
```

File: packages/satysfi-pagenumber-doc/satysfi-pagenumber-doc.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+  "satysfi-dist"
+  "satysfi-pagenumber"
```
Comment (na4zagin3) [inline packages/satysfi-pagenumber-doc/satysfi-pagenumber-doc.1.0.0/opam:16]: Can you specify the version constraint of the main library?

```suggestion
  "satysfi-pagenumber" {= "%{version}%"}
```

**Review by na4zagin3 — CHANGES_REQUESTED** [review]

File: packages/satysfi-pagenumber/satysfi-pagenumber.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A SATySFi package for number of pages"
+description: """
+Brilliant description comes here.
```
Comment (na4zagin3) [inline packages/satysfi-pagenumber/satysfi-pagenumber.1.0.0/opam:4]: Do you mind to add a short description here or remove the `description` field?

**Comment by abenori** [comment]
Thank you. I modified.

**Review by na4zagin3 — APPROVED** [review]

**Comment by na4zagin3** [comment]
LGTM.  Thank you!

**Pull request merged**